postgres-cmd.sh

```diff
@@ -0,0 +1,15 @@
+set -ex
+echo $SHELL
+echo $USER
+mkdir $DATA_DIR
```

Comment:
DATA_DIR can be fixed to `/tmp/pg_data` here since `/tmp` is a fresh tmpfs in the sandbox's mount namespace